lib/config/configuration.js

```diff
@@ -24,22 +24,130 @@ var BUILTIN_OPTIONS = {
  * @name Configuration
  */
 function Configuration() {
+    /**
+     * List of the registered (not used) presets.
+     * @protected
```

Comment:
Sorry, there is an issue for that :( https://github.com/jscs-dev/jscs-jsdoc/issues/85